heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -481,7 +521,7 @@ public FullQuery end(int resolved, int failed, int cancelled) throws Exception {
                     checkIssues(failed, cancelled).orElse("Query exceeded quota")));
 
                 return new FullQuery(trace, errorsBuilder.build(), ImmutableList.of(),
-                    Statistics.empty(), new ResultLimits(limitsBuilder.build()));
+                                     Statistics.empty(), new ResultLimits(limitsBuilder.build()));
```

Comment:
There's still a few formatting changes like this that would be cancelled out if you auto-format. So please do apply the formatter before committing.